datahub-web-react/src/app/identity/group/GroupList.tsx

```diff
@@ -49,7 +49,7 @@ export const GroupList = () => {
     const { loading, error, data, refetch, client } = useListGroupsQuery({
         variables: {
             input: {
-                start,
+                start: query && start === 0 ? null : start,
```

Comment:
can you explain the change here, please ? my original thinking is that we can reset the page back to 0 but only when the query changes... maybe using useEffect on the query variable

Reply:
There is one Scenario - While searching if we have data of  28 record of Connector( like Mongo) then  if  start 0 and page 25 it will search data between 0-25 and if we change to second page it will search 25-50. When we set start null and page size 25 it will check from all the data and give 25 record and if we click on 2nd page and not set start as 25 then it will again check from first page that the reason  I have set if Query changes and start is 0 check from all the data and when I click on 2nd page just give data which is on 2nd page. Also start is not set in state that the reason not use useEffect to set start 0 when query change. let me know if you understand otherwise we can take a call